onnxscript/rewriter/llama_rule_sets_test.py

```diff
@@ -19,7 +19,10 @@ class LlamaRuleSetsTest(unittest.TestCase):
     def _get_random_inputs(self, model: onnx.ModelProto) -> dict[str, Any]:
         feeds: dict[str, Any] = {}
         for i in model.graph.input:
-            shape = tuple(d + 2 for d in range(len(i.type.tensor_type.shape.dim)))
+            ish = tuple(i.type.tensor_type.shape.dim)
+            shape = tuple(
+                (d.dim_value if d.dim_value > 0 else i + 2) for i, d in enumerate(ish)
+            )
```

Comment:
I would improve readability of this logic: `ish`, and why the change is needed

Reply:
I added a comment. About readibility, what do you have in mind? It looks ok to me.